go/chat/search/md.go

```diff
@@ -0,0 +1,67 @@
+package search
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/keybase/client/go/protocol/chat1"
+)
+
+const indexMetadataVersion = 3
+
+type indexMetadata struct {
+	SeenIDs map[chat1.MessageID]chat1.EmptyStruct `codec:"s"`
```

Comment:
store as array on disk? some for terms/aliases

Reply:
Don't think it matters.